notifications_utils/template.py

```diff
@@ -645,14 +645,19 @@ def __init__(self, keys):
         super(NoPlaceholderForDataError, self).__init__(", ".join(keys))
 
 
-def get_sms_fragment_count(character_count, is_unicode):
-    if is_unicode:
+def get_sms_fragment_count(character_count, non_gsm_characters):
+    if len(non_gsm_characters) != 0:
```

Comment:
Optional: I believe that this could just be `if non_gsm_characters`, as it will always return a set (and the empty set is bool false).

Depends if you want to keep the `len` in particular for readability (or my above statement is false).